File: domain/src/main/java/com/aceplus/domain/model/forApi/Street.java
package com.aceplus.domain.model.forApi;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Street {

    @SerializedName("Id")
    @Expose
    private String id;
    @SerializedName("StreetNo")
    @Expose
    private String streetId;
    @SerializedName("Description")
    @Expose
    private String streetName;

    public String getStreetId() {
        return streetId;
    }

    public void setStreetId(String streetId) {
        this.streetId = streetId;
    }

    public String getStreetName() {
        return streetName;
    }

    public void setStreetName(String streetName) {
        this.streetName = streetName;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: domain/src/main/java/com/aceplus/domain/entity/predefine/Street.kt
package com.aceplus.domain.entity.predefine

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


@Entity(tableName = "street")
class Street {

    @PrimaryKey
    @ColumnInfo(name = "id")
    @SerializedName("id")
    @Expose
    var id: Int = 0

    @ColumnInfo(name = "street_id")
    @SerializedName("street_id")
    @Expose
    var street_id: Int = 0

    @ColumnInfo(name = "street_name")
    @SerializedName("street_name")
    @Expose
    var street_name: String? = null


}